src/edit/items/ItemForm.js

```diff
@@ -71,6 +71,7 @@ import NoteFields from '../noteFields';
 
 import styles from './ItemForm.css';
 import { itemStatusesMap } from '../../constants';
+import AdditionalCallNumbersItemLevelFields from './repeatableFields/AdditionalCallNumbersItemLevelFields';
```

Comment:
move it right after `NoteFields` import, so css and utils imports are separated at the bottom